src/core/flexible_hvac.jl

```diff
@@ -144,78 +161,161 @@ end
 
 
 """
-    FlexibleHVAC(dict_from_json::Dict)
+    FlexibleHVAC(
+        doe_reference_name::String,
+        city::String,
+        installed_cost::Float64,
+        temperature_upper_bound_degC_heating::Real,
+        temperature_lower_bound_degC_heating::Real,
+        temperature_upper_bound_degC_cooling::Real,
+        temperature_lower_bound_degC_cooling::Real,
+    )
+
+Constructor for `FlexibleHVAC` when using a built-in RC model that has been fit to a DoE Commercial 
+Reference Building.
 
+The `city` value is optional and if not provided its value is inferred from the `Site.latitude` and
+`Site.longitude` to determine the representative city for the climate zone. 
+See the [ElectricLoad](@ref) docs for `city` options.
+
+
+!!! note
```

Comment:
Should these docs be updated to reflect _cooling or _heating suffixes to say either `temperature_upper_bound_degC_cooling` or `temperature_lower_bound_degC_heating` are required inputs?